rootfs/api/models/certificate.py

```diff
@@ -173,7 +173,7 @@ def attach_in_kubernetes(self, domain):
         # only create if it exists - We raise an exception when a secret doesn't exist
         try:
             name = '%s-cert' % self.name
-            app = domain.app
+            app = domain.app.id
```

Comment:
should probably change this to `app_id` instead of `app` so we're not confused what this variable holds in future PRs